test/testchunkingng.cpp

```diff
@@ -62,25 +65,27 @@ private slots:
 
     void testFileUpload() {
         FakeFolder fakeFolder{FileInfo::A12_B12_C12_S12()};
+        fakeFolder.syncEngine().account()->setCapabilities({ { "dav", QVariantMap{ {"chunking", "1.0"} } } });
```

Comment:
@TheOneRing is this not needed anymore?

Reply:
yes it was removed see https://github.com/owncloud/client/blob/master/test/testutils/testutils.cpp#L35 and https://github.com/owncloud/client/blob/master/test/testutils/testutils.cpp#L35